katrain/gui/badukpan.py

```diff
@@ -61,6 +62,14 @@ def __init__(self, **kwargs):
         self.bind(size=self.redraw_trigger, pos=self.redraw_trigger)
         Clock.schedule_interval(self.animate_pv, 0.1)
 
+    def enable_coordinates(self, enabled):
+        self.draw_coords_enabled = enabled
+        self.draw_board()
```

Comment:
just make this a toggle function without parameters since it's not being called any other way
call the redraw trigger rather than function (this stops holding down k being really unpleasant)